pytorch_lightning/utilities/distributed.py

```diff
@@ -381,7 +381,9 @@ def init_dist_connection(
     world_size = world_size if world_size is not None else cluster_environment.world_size()
     os.environ["MASTER_ADDR"] = cluster_environment.main_address
     os.environ["MASTER_PORT"] = str(cluster_environment.main_port)
-    if torch.distributed.is_available() and not torch.distributed.is_initialized():
+    if not torch.distributed.is_available():
+        raise RuntimeError("Torch distributed is not avalible, can not initialized distributed process group")
```

Comment:
```suggestion
        raise RuntimeError("torch.distributed is not available. Cannot initialize distributed process group")
```

Reply:
Could you apply ananthsub's suggestion again? It seems like your force-pushing reverted the change here.